Day3Part2.js

```diff
@@ -0,0 +1,53 @@
+const readline = require("readline").createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+readline.question(`enter your data `, function LifeSupportRating(data) {
+  var Data = data.split(" ");
+  zeroCounter = 0;
+  oneCounter = 0;

```

Comment:
This is how you write this line correctly:

```suggestion
  let zeroCounter = 0;
  let oneCounter = 0;
```

But it shouldn't even exist. The two variables should be created in `getMostCommon`